lib/kvdb/kvdb_home.c

```diff
@@ -70,10 +66,7 @@ kvdb_home_storage_path_get(
 
 merr_t
 kvdb_home_storage_realpath_get(
-    const char * home,
-    const char * path,
-    char         buf[PATH_MAX],
-    bool         resolved_path)
+        const char *home, const char *path, char buf[PATH_MAX], bool resolved_path)
```

Comment:
How do you guys want to format things like this?

Reply:
`AllowAllParametersOfDeclarationOnNextLine: true` fixes this.